From 2e9dc29a6d617f7e445ca7c32f16e513032125ac Mon Sep 17 00:00:00 2001
From: Rajkumar Ayyasamy <arajkuma@codeaurora.org>
Date: Wed, 20 Oct 2021 17:58:18 +0530
Subject: [PATCH] arch: arm64: add support to select diff memory profile

This change will add support to select different memory
profile for 64-bit kernel.

Change-Id: I25edf54beff3c07763c537862b1607aadd3d5d08
Signed-off-by: Rajkumar Ayyasamy <arajkuma@codeaurora.org>
Signed-off-by: Saahil Tomar <quic_saahtoma@quicinc.com>
---
 arch/arm64/Kconfig | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/Kconfig b/arch/arm64/Kconfig
index c58ea5d6db1f..7adf2f0d4f13 100644
--- a/arch/arm64/Kconfig
+++ b/arch/arm64/Kconfig
@@ -2196,6 +2196,15 @@ config STACKPROTECTOR_PER_TASK
 	def_bool y
 	depends on STACKPROTECTOR && CC_HAVE_STACKPROTECTOR_SYSREG
 
+config IPQ_MEM_PROFILE
+	int "Select Memory Profile"
+	range 0 1024
+	default 0
+	help
+	  This option select memory profile to be used,which defines
+	  the reserved memory configuration used in device tree.
+	  If unsure, say 0
+
 config UNWIND_PATCH_PAC_INTO_SCS
 	bool "Enable shadow call stack dynamically using code patching"
 	# needs Clang with https://reviews.llvm.org/D111780 incorporated
-- 
2.34.1

